Prox/Prox/PlaceDetails/PlaceDetailViewController.swift

```diff
@@ -185,6 +217,12 @@ class PlaceDetailViewController: UIViewController {
         UIView.animate(withDuration: animationDuration, delay: 0.0, usingSpringWithDamping: springDamping, initialSpringVelocity: 0.0, options: .curveEaseOut, animations: {
             self.imageCarousel.alpha = 0
             nextCardImageCarousel.alpha = 1
+            if let imageURLString = nextCardViewController.place.photoURLs?.first,
```

Comment:
nit: move this to a helper function and call it from here and for `pageToPrevious` (line 316)